src/CppUTest/SimpleString.cpp

```diff
@@ -478,6 +478,11 @@ SimpleString HexStringFrom(long value)
     return StringFromFormat("%lx", value);
 }
 
+SimpleString HexStringFrom(int value)
+{
+    return StringFromFormat("%x", value);
+}
+
 SimpleString HexStringFrom(signed char value)
 {
     SimpleString result = StringFromFormat("%x", value);
```

Comment:
@arstrube ,  I checked the merge and it went well. The above codes slices only display the difference, and it don't display the whole codes, the codes
```.cpp
SimpleString HexStringFrom(signed char value)
{
    SimpleString result = StringFromFormat("%x", value);
    if(value < 0) {
        size_t size = result.size();
        result = result.subString(size-(CPPUTEST_CHAR_BIT/4));
    }
    return result;
}
```
still there. 

You can fetch my branch and do a diff to see :)